src/connection-manager/auto-dialler.ts

```diff
@@ -48,6 +52,8 @@ export class AutoDialler implements Startable {
     this.options = mergeOptions.call({ ignoreUndefined: true }, defaultOptions, init)
     this.running = false
     this._autoDial = this._autoDial.bind(this)
+    this.abortController = new AbortAbort()
```

Comment:
Why not just use a regular `AbortController` here?

Create it in `.start`, pass it's `.signal` to every dial attempt in `._autoDial` and `.abort()` it in `.stop`.